### PR TITLE
Docs: Add link to Discourse Forum

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Adds link to the "dashboard-2" topic on the Node-RED forums.

### CODE REVIEW

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export default ({ mode }) => {
         { text: 'Docs', link: '/getting-started.html' },
         { text: 'Widgets', link: '/nodes/widgets.html' },
         { text: 'Contributing', link: '/contributing/' },
+        { text: 'Forum', link: 'https://discourse.nodered.org/tag/dashboard-2' },
```
Comment (ZJvandeWeg) [inline docs/.vitepress/config.js:57]: Can we also include the discord for chat?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: docs/.vitepress/config.js (hunk 1 above)
Comment (Paul-Reed) [reply docs/.vitepress/config.js:57]: Shouldn't all the documentation really be in one place instead of users having to follow links to other sites...

Comment (joepavitt) [reply docs/.vitepress/config.js:57]: The documentation is all available in the docs, but this links to the Forum for discussion of topics, or to the Discord for those that are operating there instead (as proposed by ZJ)

Comment (joepavitt) [reply docs/.vitepress/config.js:57]: RE: Discord link specifically, I'm a little cautious of over-loading that top-right corner, but can add it if required... doesn't stop this PR though which is constrained just to the "Forum".

Adding a Discord link should be a new PR imo

**Comment by Paul-Reed** [comment]
Sorry Joe, my comment should have gone to #724 and not here (working via
phone!)

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
@joepavitt The E2E is failing, can you take a look?

**Comment by joepavitt** [comment]
> The E2E is failing, can you take a look?

Known problem with inconsistencies in the websocket traffic when running in GH

**Pull request merged**